### PR TITLE
Some performance issue fix.

### PR DESCRIPTION
**Pull request by luodanwg**
Some performance issue fix.

<dbftService.go>
1.When in BlockSent status, should not changeView, just need to wait the block persistent complete.
<blockValidator.go>
1.append has a large performance loss,so changed to only 2 append process.
2.should not return when  VerifySignableData ok, otherwise it will skip the transaction verify.
<txValidator.go>
1.With no UTXOInputs could return it immediately. otherwise, it will run len([]*tx.Transaction) times.

Signed-off-by: luodanwg<luodan.wg@gmail.com>

### CODE REVIEW

File: core/validation/blockValidator.go [hunk 1]
```diff
@@ -19,7 +19,6 @@ func VerifyBlock(block *ledger.Block, ld *ledger.Ledger, completely bool) error
 
 	flag, err := VerifySignableData(block)
 	if flag && err == nil {
-		return nil
```
Comment (luodanwg) [inline core/validation/blockValidator.go]: when VerifySignableData ok , it will skip the below process.

File: core/validation/txValidator.go [hunk 2]
```diff
@@ -96,7 +96,9 @@ func VerifyTransaction(Tx *tx.Transaction, ledger *ledger.Ledger, TxPool []*tx.T
 }
 
 func CheckMemPool(tx *tx.Transaction, TxPool []*tx.Transaction) error {
-
+	if len(tx.UTXOInputs)==0{
```
Comment (dreamfly281) [inline core/validation/txValidator.go]: "if len(tx.UTXOInputs)==0{"  should be follow the format  "if len(tx.UTXOInputs) == 0 {" which is the default "go fmt" style, it is better run "go fmt" before submit the code

File: core/validation/txValidator.go [hunk 3]
```diff
@@ -110,6 +112,9 @@ func CheckMemPool(tx *tx.Transaction, TxPool []*tx.Transaction) error {
 }
 
 func CheckDuplicateInput(tx *tx.Transaction) error {
+	if len(tx.UTXOInputs)==0{
+		return nil
```
Comment (dreamfly281) [inline core/validation/txValidator.go]: ditto

File: core/validation/blockValidator.go (hunk 1 above)
Comment (dreamfly281) [inline core/validation/blockValidator.go]: if the "return nil" be removed, the if .. else just have no meaning here, so just adjust the code to the if condition like below 

if  flag == false  || err != nil {
    return err 
}

Comment (luodanwg) [inline core/validation/blockValidator.go]: Thanks for your review. could you helps to check it again. thanks.

File: core/validation/txValidator.go (hunk 2 above)
Comment (luodanwg) [inline core/validation/txValidator.go]: Thanks for your review. could you helps to check it again. thanks.

File: core/validation/txValidator.go (hunk 3 above)
Comment (luodanwg) [inline core/validation/txValidator.go]: Thanks for your review. could you helps to check it again. thanks.

**Pull request merged**